src/app/upload-popup/upload-popup.component.html

```diff
@@ -0,0 +1,21 @@
+<div id="image-banner">
+  <h2 mat-dialog-title>Upload Image</h2>
+  <button id="close" mat-icon-button (click)="close()">
+    <mat-icon>close</mat-icon>
+  </button>
+</div>
+<mat-dialog-content *ngIf="data!=&quot;&quot;">
+  <img class="preview-image" src={{this.data}}>
+</mat-dialog-content>
+<mat-dialog-actions>
+  <input id="selected-image" #selectedImage
+    *ngIf="flashcard.definition.length <= 72; else stopMessage"(change)="uploadImage(selectedImage)"
```

Comment:
could you make 72 a variable with a descriptive name? Repeat this for other hard coded values